### PR TITLE
Add GetSong With Lyrics API

### PR DESCRIPTION
**Pull request by brandonxia01**
1. Add get songs with lyrics API. 

Screen Shot 2023-02-22 at 2 47 44 PM

2. Add APIs to insert song and insert lyrics.
3. Added Test cases

4. I also wrote a script which runs through the SHL songbook and populates the database with all the songs and lyrics.
https://github.com/Church-Life-Apps/Utilities/blob/main/scripts/populate_database_for_shl_songbook.py

### CODE REVIEW

File: src/test/utils/StringUtils.test.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// import {isNumeric} from '../../utils/StringUtils';
+
+// test('Tests IsNumeric', () => {
```
Comment (brandonxia01) [inline src/test/utils/StringUtils.test.ts:3]: Test cases will come soon... I tried for like 20mins to set up Jest but I don't know how. May need help.

File: src/index.ts [hunk 2]
```diff
@@ -22,33 +24,79 @@ app.use(express.static(path.resolve(__dirname, '../web-build')));
...
+  res.status(200).send(await songsService.getSongbooks());
 });
 
 // List Songs API
 app.get('/songs', async (req, res) => {
   try {
     const songbookId = (req.query.songbookId as string) ?? '';
     validateGetSongsRequest(songbookId);
-    res.status(200).send(await getSongsMethod(songbookId));
+    res.status(200).send(await songsService.getSongsMethod(songbookId));
```
Comment (MiTo0o) [inline src/index.ts:43]: for all of the `res.status(200).send()` you can just write `res.send()` because status code 200 is the default.

**Review by MiTo0o — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 3]
```diff
@@ -22,33 +24,79 @@ app.use(express.static(path.resolve(__dirname, '../web-build')));
...
+
 app.get('/hi', (req, res) => {
   res.status(200).send('hello');
 });
 
 // List Songbooks API
 app.get('/songbooks', async (_req, res) => {
-  res.status(200).send(await querySongbooks());
+  res.status(200).send(await songsService.getSongbooks());
 });
```
[the remark this answers is not in the document]
Comment (brandonxia01) [reply src/index.ts]: Will do!

File: src/index.ts (hunk 2 above)
Comment (brandonxia01) [reply src/index.ts]: Cool

File: src/services/SongsService.ts [hunk 4]
```diff
@@ -1,18 +1,39 @@
...
-  return await insertSongbook(songbook);
-}
+import {DbLyric, DbSong, DbSongbook, DbSongWithLyrics} from '../db/DbModels';
+import {SongsDb} from '../db/SongsDb';
+
+export class SongsService {
+  songsDb: SongsDb;
+
+  constructor() {
+    const songsDb = new SongsDb(null);
```
[the remark this answers is not in the document]
Comment (brandonxia01) [reply src/services/SongsService.ts]: Good catch!

**Review by MiTo0o — APPROVED** [review]
lgtm

**Pull request merged**